.github/workflows/ci.yaml

```
@@ -8,7 +8,7 @@ on:
   pull_request: ~
 
 env:
-  CACHE_VERSION: 4
+  CACHE_VERSION: 4 # Also change this in the primer workflow
```

```suggestion
  CACHE_VERSION: 4  # Also change this in the primer workflow
```
It's usually common to put 2 whitespaces between code and a comment, same as in Python.